RHEL6 e2e capabilities test failure

### Version of Singularity:

What version of Singularity are you using? Run:

```
3.5.2
```

### Actual behavior

```
                --- FAIL: TestE2E/PAR/SECURITY/singularitySecurityPriv/capabilities_keep (0.14s)
                    security.go:186: Running command "/usr/local/bin/singularity --debug exec --keep-privs /tmp/stest.044092907/test.sif grep ^C
apEff: /proc/self/status"
                    security.go:186: failed to match pattern: Command "/usr/local/bin/singularity --debug exec --keep-privs /tmp/stest.044092907/test.sif grep ^CapEff: /proc/self/status":
                        Expect output stream match regular expression:
                        CapEff:\s+0000003fffffffff\n
                        Command output output:
                        CapEff: ffffffffffffffff
                        
                        github.com/sylabs/singularity/e2e/internal/e2e.(*SingularityCmdResult).expectMatch
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:124
                        github.com/sylabs/singularity/e2e/internal/e2e.ExpectOutput.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:140
                        github.com/sylabs/singularity/e2e/internal/e2e.ExpectExit.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:435
                        github.com/sylabs/singularity/e2e/internal/e2e.TestEnv.RunSingularity.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:633
                        github.com/sylabs/singularity/e2e/internal/e2e.Privileged.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/priv_linux.go:53
                        testing.tRunner
                                testing/testing.go:909
                        runtime.goexit
                                runtime/asm_amd64.s:1357
                        matching "CapEff:\\s+0000003fffffffff\\n" of type RegexMatch in output stream
                        github.com/sylabs/singularity/e2e/internal/e2e.ExpectOutput.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:141
                        github.com/sylabs/singularity/e2e/internal/e2e.ExpectExit.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:435
                        github.com/sylabs/singularity/e2e/internal/e2e.TestEnv.RunSingularity.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:633
                        github.com/sylabs/singularity/e2e/internal/e2e.Privileged.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/priv_linux.go:53
                        testing.tRunner
                                testing/testing.go:909
                        runtime.goexit
                                runtime/asm_amd64.s:1357
                --- FAIL: TestE2E/PAR/SECURITY/singularitySecurityPriv/capabilities_drop (0.14s)
                    security.go:186: Running command "/usr/local/bin/singularity --debug exec --drop-caps CAP_NET_RAW /tmp/stest.044092907/test.sif grep ^CapEff: /proc/self/status"
                    security.go:186: failed to match pattern: Command "/usr/local/bin/singularity --debug exec --drop-caps CAP_NET_RAW /tmp/stest.044092907/test.sif grep ^CapEff: /proc/self/status":
                        Expect output stream match regular expression:
                        CapEff:\s+0000003fffffdfff\n
                        Command output output:
                        CapEff: ffffffffffffdfff
                        
                        github.com/sylabs/singularity/e2e/internal/e2e.(*SingularityCmdResult).expectMatch
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:124
                        github.com/sylabs/singularity/e2e/internal/e2e.ExpectOutput.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:140
                        github.com/sylabs/singularity/e2e/internal/e2e.ExpectExit.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:435
                        github.com/sylabs/singularity/e2e/internal/e2e.TestEnv.RunSingularity.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:633
                        github.com/sylabs/singularity/e2e/internal/e2e.Privileged.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/priv_linux.go:53
                        testing.tRunner
                                testing/testing.go:909
                        runtime.goexit
                                runtime/asm_amd64.s:1357
                        matching "CapEff:\\s+0000003fffffdfff\\n" of type RegexMatch in output stream
                        github.com/sylabs/singularity/e2e/internal/e2e.ExpectOutput.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:141
                        github.com/sylabs/singularity/e2e/internal/e2e.ExpectExit.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:435
                        github.com/sylabs/singularity/e2e/internal/e2e.TestEnv.RunSingularity.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:633
                        github.com/sylabs/singularity/e2e/internal/e2e.Privileged.func1
```


### What OS/distro are you running

```
RHEL6
```

--- a/e2e/security/security.go
+++ b/e2e/security/security.go
@@ -157,7 +157,7 @@ func (c ctx) testSecurityPriv(t *testing.T) {
 			name:     "capabilities_keep",
 			argv:     []string{"grep", "^CapEff:", "/proc/self/status"},
 			opts:     []string{"--keep-privs"},
-			expectOp: e2e.ExpectOutput(e2e.RegexMatch, `CapEff:\s+0000003fffffffff\n`),
+			expectOp: e2e.ExpectOutput(e2e.RegexMatch, `CapEff:\s+[0f]{6}[3f]fffffffff\n`),
 		},
 		{
 			// this might break if new capabilities are
@@ -171,7 +171,7 @@ func (c ctx) testSecurityPriv(t *testing.T) {
 			name:     "capabilities_drop",
 			argv:     []string{"grep", "^CapEff:", "/proc/self/status"},
 			opts:     []string{"--drop-caps", "CAP_NET_RAW"},
-			expectOp: e2e.ExpectOutput(e2e.RegexMatch, `CapEff:\s+0000003fffffdfff\n`),
+			expectOp: e2e.ExpectOutput(e2e.RegexMatch, `CapEff:\s+[0f]{6}[3f]fffffdfff\n`),
 		},
 	}
 

